### PR TITLE
[Feature] World downloader

### PR DESCRIPTION
**Pull request by hax0r31337**

### CODE REVIEW

**Comment by AyanoYuugiri** [comment]
sorry i totally forgot about it

**Review by AyanoYuugiri — APPROVED** [review]
seems working fine

File: src/main/java/dev/sora/relay/game/world/leveldb/LevelDBWorld.kt [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+import org.iq80.leveldb.impl.Iq80DBFactory
+import java.io.File
+
+/**
+ * From PowerNukkitX
+ * @author Superrice666, NekoRibbon
+ */
+class LevelDBWorld(val folder: File) {
+
+	private val db = Iq80DBFactory.factory.open(folder, Options().createIfMissing(true).compressionType(CompressionType.ZLIB_RAW))
```
Comment (AyanoYuugiri) [inline src/main/java/dev/sora/relay/game/world/leveldb/LevelDBWorld.kt:17]: The compression seems not working as expected

**Review by AyanoYuugiri — CHANGES_REQUESTED** [review]

File: src/main/java/dev/sora/relay/game/world/leveldb/LevelDBWorld.kt (hunk 1 above)
Comment (hax0r31337) [reply src/main/java/dev/sora/relay/game/world/leveldb/LevelDBWorld.kt:17]: idk how 2 make the compression working ;w;

**Pull request merged**